packages/jaeger-ui/src/components/DependencyGraph/DAG.test.js

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { shallow } from 'enzyme';
```

Comment:
we are trying to move away from enzyme, please do not reintroduce it here, use testing-library

Reply:
especially considering that the tests here are extremely primitive, just a check for top-level element

Reply:
There's a comment in the test file that we don't really care about the rendered canvas actually, we still can check for the number of nodes and edges to be more precise